lib/libmbedtls/core/des3_cmac.c

```diff
@@ -0,0 +1,159 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (C) 2018, ARM Limited
+ * Copyright (C) 2019, Linaro Limited
+ */
```

Comment:
Add your own copyright perhaps?

Reply:
Sure, I'll add a copyright of my current employer then as this was initially created for internal test project.
btw, I also have a des-perf command added to xtest (based heavily(surprise) on aes-perf) so I can push it later as well if you think it makes sense.

Reply:
This file seems to be just a copy of aes_cmac.c with only a few essential changes. It looks like it should be possible to shared quite a bit of code instead of duplicating it. Compare with the LTC case.

Reply:
Yes, this is the case.
If you are OK with it I can do it in this PR with a nice bonus of handling panic()/assert() for AES CMAC.

Reply:
Perhaps I merged this a bit too quickly :open_mouth: sorry about that, but this kind of refactoring can be made in a separate PR with no problem.

Reply:
@jforissier @jenswi-linaro no worries, I'll handle this in the next PR:)